cmd/future/bump_deps.go

```diff
@@ -27,7 +28,20 @@ var BumpDeps = &cobra.Command{
 
 		if err := composer.WriteComposerJson(file, s); err != nil {
 			log.Fatalf("could not write composer.json: %v\n", err)
+			return
+		}
+
+		if len(deps.Installed) == 0 {
+			log.Print("all dependencies are at their latest version - nothing to update\n")
```

Comment:
these will eventuall change into gRPC calls to our summary process, but for now let's keep them this way so we know what's up